spec/lib/roadie/document_spec.rb

```diff
@@ -98,6 +104,15 @@ module Roadie
         }
         expect { document.transform }.to_not raise_error
       end
+
+      context 'when save as xhtml flag is present' do
+        it "does not escape some HTML symbols" do
+          document = Document.new "<body><a href='https://google.com/{{hello}}'</body>"
```

Comment:
Handlebars using `{{hello}}` to replace the variable with passed locales. In this case this could be everything.